src/core/dbus-manager.c

```diff
@@ -2146,7 +2146,7 @@ static int install_error(
 
                 /* When making changes here, make sure to also change install_changes_dump() in install.c. */
 
-                switch (changes[i].type) {
+                switch (changes[i].change_or_error) {
```

Comment:
you are reverting f8662fee2fac735385dc060f1435e4af5327138d with that. At the very least, say so.

Reply:
f8662fee2fac735385dc060f1435e4af5327138d actually reverts (part of) 1308f72e831dcb634e7798c125611808f8b76cff. Frankly, I didn't look at the history, and thought that I dropped the rename from that pull request somehow, as it underwent a lot of review and reworks, and I ended up dropping at least one patch. But as I wrote in #25002, this is getting into bikeshed territory so I'll drop this commit instead.